drivers/gnss/quectel_lcx6g.c

```diff
@@ -0,0 +1,754 @@
+/*
+ * Copyright (c) 2023 Trackunit Corporation
+ * Copyright (c) 2023 Bjarki Arge Andreasen
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/gnss.h>
+#include <zephyr/modem/chat.h>
+#include <zephyr/modem/backend/uart.h>
+#include <zephyr/kernel.h>
+#include <zephyr/pm/device.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/pm/device_runtime.h>
+#include <string.h>
+
+#include "gnss_publish.h"
+#include "gnss_nmea0183.h"
+#include "gnss_parse.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(quectel_lcx6g, CONFIG_GNSS_LOG_LEVEL);
```

Comment:
would probably be worthwhile to _actually_ add some logging to the driver as it might help folks troubleshoot/understand better?

Reply:
I will look into it :)

Reply:
Behold https://github.com/zephyrproject-rtos/zephyr/compare/19daeb4d3647e02021a8c2431172d20315b6f154..fb379779dd90a4d6869c581668eb4fc8d9dacf98